R/sample.R

```diff
@@ -145,6 +201,7 @@ combine_chains <- function(res) {
   density <- unlist(lapply(res, "[[", "density"))
   details <- lapply(res, "[[", "details")
   details <- if (all(vlapply(details, is.null))) NULL else details
+  rng_state <- lapply(res, function(x) x$internal$rng_state)
```

Comment:
Should this get used yet?

Reply:
vestigial I think, from before I tidied up the restart data